odonto/odonto_submissions/tests/test_serializors.py

```diff
@@ -570,16 +581,10 @@ def test_get_bcds1(self, translate_to_bdcs1):
         self.user.save()
 
         bcds1 = serializers.get_bcds1(episode, "REF_NUM", "SUB_COUNT")
-        self.assertEqual(bcds1.contract_number, "FP17_CONTRACT_NUMBER")
```

Comment:
This is now done in the translate_to_bdcs1 method which is mocked. It is tested in the test cases.